python/cudnn/csa/__init__.py

```
@@ -0,0 +1,36 @@
+
+
+class CSANamespace:
+    def __getattr__(self, name):
+        if name in _SYMBOLS:
+            return _load_symbol(name)
+        raise AttributeError(f"CSA has no attribute {name!r}")
+
+
+CSA = CSANamespace()
+
+__all__ = ["CSA", *_SYMBOLS.keys()]
```

Ruff (0.15.21)

[error] Invalid object in `__all__`, must contain only strings

(PLE0604)